app/index.html

```diff
@@ -157,6 +157,11 @@
               <img src="http://gravatar.com/avatar/48fa294e3cd41680b80d3ed6345c7b4d?s=144" alt="Kevin Mårtensson">
               <span class="member">Kevin Mårtensson</span>
             </a>
+          </li><li>
+            <a href="https://github.com/arthurvr">
+              <img src="https://secure.gravatar.com/avatar/e34daab0d2e344219adb5234198269c5?size=144" alt="Arthur Verschaeve">
```

Comment:
URL should be: `https://gravatar.com/avatar/e34daab0d2e344219adb5234198269c5?s=144`